torchci/pages/flakytest.tsx

```diff
@@ -59,12 +57,11 @@ export default function Page() {
                   <td className={styles.table}>{test.name}</td>
                   <td className={styles.table}>{test.suite}</td>
                   <td className={styles.table}>{test.file}</td>
-                  <td className={styles.table}>{test.numGreen}</td>
-                  <td className={styles.table}>{test.numRed}</td>
                   <td className={styles.table}>
-                    {convertWorkflowIDtoURLs(test.workflowIds).join("\n")}
+                    {convertJobIDtoURLs(test.jobIds).join("\n")}
```

Comment:
can we convert these to <a> tags so it's clickable? Also the github.com... is a bit repetitive so you might want to get rid of those as well